docs/docs/gatsby-link.md

```diff
@@ -89,28 +89,21 @@ The `activeStyle` or `activeClassName` prop are only set on a `<Link>` component
 - We may want `/blog/hello-world` to match `<Link to="/blog">`
 - Or `/gatsby-link/#passing-state-through-link-and-navigate` to match `<Link to="/gatsby-link">`
 
-In instances like these, we can use [@reach/router's](https://reach.tech/router/api/Link) `getProps` API to to set active styles like in the following example:
+In instances like these, we can use [@reach/router's](https://reach.tech/router/api/Link) `getProps` API to add the string `"active"` to the `className` of only partially active links as follows:
```

Comment:
Let's maybe revert this? I don't think this level of detail is needed.

Reply:
Sure thing.